Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/StructurePanel.java

```diff
@@ -1114,4 +1122,114 @@ private IncludedStructuralElement getTreeNodeStructuralElement(TreeNode treeNode
         }
         return null;
     }
+
+    /**
+     * Get List of MediaUnits assigned to multiple IncludedStructuralElements.
+     *
+     * @return value of severalAssignments
+     */
+    List<MediaUnit> getSeveralAssignments() {
+        return severalAssignments;
+    }
+
+
+    /**
+     * Get the index of this StructureTreeNode's MediaUnit out of all MediaUnits
+     * which are assigned to more than one IncludedStructuralElement.
+     *
+     * @param treeNode object to find the index for
+     * @return index of the StructureTreeNode's MediaUnit if present in the List of several assignments, or -1 if not present in the list.
+     */
+    public int getMultipleAssignmentsIndex(StructureTreeNode treeNode) {
+        if (Objects.nonNull(treeNode.getDataObject()) && treeNode.getDataObject() instanceof View
```

Comment:
```suggestion
        if (treeNode.getDataObject() instanceof View
```

instanceof already checks against null